nixos/doc/manual/release-notes/rl-2211.section.md

```diff
@@ -54,6 +54,8 @@ In addition to numerous new and upgraded packages, this release has the followin
 
 - [appvm](https://github.com/jollheef/appvm), Nix based app VMs. Available as [virtualisation.appvm](options.html#opt-virtualisation.appvm.enable).
 
+- [cups-pdf-to-pdf](https://github.com/alexivkin/CUPS-PDF-to-PDF), a pdf-generating cups backend based on [cups-pdf](https://www.cups-pdf.de/).  Available as [services.printing.cups-pdf](#opt-services.printing.cups-pdf.enable).
```

Comment:
I don't think it makes sense to install `cups-pdf` and `cups-pdf-to-pdf` alongside on the same system: They are looking for their config files in the same place, so they couldn't be configured independently.  Hence I would refrain from ever adding two nixos options or modules for those packages.

(Of course we can patch the package to look for a different path, but I guess that would be a bit surprising for package users who do not use the NixOS module.)

If we want to provide the original `cups-pdf` package as an alternative, we can simply add a `package` option where the user can pick the favored package, something like

```nix
{
  type = lib.types.package;
  default = pkgs.cups-pdf-to-pdf;
  example = lib.literalExample "pkgs.cups-pdf";
}
```

to switch to the other package.  The NixOS module would then use this package to perform its suid wrapper magic.

However, since `cups-pdf` is a well-known name (at least, that's my feeling), and since `cups-pdf-to-pdf` is just a fork, I would still call this option `cups-pdf` and hope it leads to less confusion.